artifacts/ts/contracts.ts

```diff
@@ -0,0 +1,42 @@
+/* Autogenerated file. Do not edit manually. */
+/* tslint:disable */
+/* eslint-disable */
+
+import { Contract, ContractFactory } from "@alephium/web3";
+import {
+  Add,
+  Greeter,
+  NFTCollectionTest,
+  NFTTest,
+  Sub,
+  Assert,
+  Debug,
+  MetaData,
+  Warnings,
+  FakeTokenTest,
+  TokenTest,
+} from ".";
+
+export function getContractByCodeHash(codeHash: string): Contract {
+  const contracts: ContractFactory<any>[] = [
```

Comment:
It doesn't work if move the `contracts` outside the function.

Reply:
It should work if we use arrow function to capture the closure:
```
const contracts = ...

export const getContractByCodeHash = (codeHash: string): Contract => {
  ...
}
```